.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,70 @@
+---
+name: ➕ Feature request
+description: Suggest an idea for setuptools
+title: 'Feature: '
+labels:
+- enhancement
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to suggest a feature for XLConnect!**

```

```suggestion
      **Thank you for suggesting a feature for XLConnect**

```
Align to `bug-report.yml`.